src/llm_vm/onsite_llm.py

```diff
@@ -66,7 +67,99 @@ def generate(self): # this is where the meat and potatoes should live?
 this factorization isn't necessarily the greatest, nor should it be viewed
 as likely being more general, aside from covering hugging face transformers
 """
+    
+class Small_Local_Pythia:
+    """
+    This is a class for ElutherAI's Pythia-70m LLM
+
+    Attributes:
+        model_uri (str): Hugging Face Endpoint for LLM
+        tokenizer (AutoTokenizer): Tokenizer from Transformer's library
+        model (LLM): The large language model
+
+    Methods:
+        model_loader: Loads the LLM into memory
+        tokenizer_loader: Loads the tokenizer into memory
+        generate: Generates a response from a given prompt with the loaded LLM and tokenizer
+    """
+    
+    def __init__(self,model_uri_override="EleutherAI/pythia-70m-deduped"): # tokenizer_kw_args=None,model_kw_args=None
```

Comment:
We should remember to rename this and maybe merge in the partial doc branch?

Reply:
I changed the name to model_uri, we'll need to recode the workflow for these classes to incorporate parameter setting for different sized models.